tools/sgen/input/FluentUIStyle.yml

```diff
@@ -445,6 +444,19 @@ AP_MSFListCellTokens:
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
+  
+AP_MSFHeaderFooterTokens:
+  backgroundColor: [ default: $Colors.Background.neutral1, divider: $Colors.Background.neutral5 ]
```

Comment:
Seems like both height and background color only use the default property:


```
backgroundColor = appearanceProxy.backgroundColor.default
headerHeight = appearanceProxy.headerHeight.default
```

Reply:
Re: the divider has different styles so I wanted to add them in ahead of time

Reply:
As per our offline conversation, we decided not to add stuff that the code is not using right now.